packages/components/src/ItineraryCard/TripSector.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Text, StyleSheet, CarrierLogo } from '@kiwicom/universal-components';
+import * as DateFNS from 'date-fns';
+import { uniq } from 'ramda';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import type { TripSector as TripSectorType } from './__generated__/TripSector.graphql';
+import TimelineArrow from './TimelineArrow';
+
+type Props = {|
+  +data: ?TripSectorType,
+|};
+const timeSimpleFormat = 'H:mm';
+const dateFormat = 'ddd D MMM';
+
+const getFormattedDate = (time, format = timeSimpleFormat) =>
+  DateFNS.format(DateFNS.parse(time), format);
+
+const getDuration = durationInMinutes => {
+  return (
+    durationInMinutes &&
+    `${Math.floor(durationInMinutes / 60)}h ${durationInMinutes % 60}m`
```

Comment:
This is not going to be translation friendly. Can DateFNS deal with it somehow?

Reply:
True, I will check it

Reply:
Fixed